src/xhr/components/content.md

```diff
@@ -1 +1,20 @@
 The section houses Xhr components for making HTTP requests and handling server errors gracefully.
+
+**_NOTE: Consumers will need to install wathwg-fetch pollyfill to work in IE_**
```

Comment:
I discovered we polyfill this in [monorepo](https://github.com/getethos/ethos/blob/trunk/frontend/shared/webApp/polyfill.js#L7) already, but not yet in Nora. I believe Tim was well aware of this as he added the polyfill and was original author of XHR, but we hadn't had a chance to discuss. So I think this is an important notice for consumers to see given the nuance that Babel does not give you this for free!

Reply:
* whatwg-fetch

Reply:
thanks, fixed in 5b9376e